src/posthog-featureflags.ts

```diff
@@ -405,9 +410,9 @@ export class PostHogFeatureFlags {
         }
     }
 
-    _fireFeatureFlagsCallbacks(): void {
+    _fireFeatureFlagsCallbacks(errorsLoading?: boolean): void {
         const { flags, flagVariants } = this._prepareFeatureFlagsForCallbacks()
-        this.featureFlagEventHandlers.forEach((handler) => handler(flags, flagVariants))
+        this.featureFlagEventHandlers.forEach((handler) => handler(flags, flagVariants, errorsLoading))
```

Comment:
I really have a pet peeve for functions that just have a bunch of arguments that can be `true` or `1` or something.

Feels like this is meta info - why not turn it into an object like `context: { errors: true }` so that it would be easy to extend further in the future.

Reply:
Very reasonable, will do 👍

Reply:
Makes sense for the public interface to have a context object, but keeping interals as a param. Don't see any need to convert yet, and easy to refactor when/if we have more params to these internal funcs